### ISSUE TITLE
SystemExit

### ISSUE THREAD

**Issue by allaabdella2** [altosaar/variational-autoencoder#12]
Hi Altosaar, 

I run your code on collab.research.google.com and it gave this error:
"SystemExit
/usr/local/lib/python3.6/dist-packages/IPython/core/interactiveshell.py:2890: UserWarning: To exit: use 'exit', 'quit', or Ctrl-D.
  warn("To exit: use 'exit', 'quit', or Ctrl-D.", stacklevel=1)"

Can you please run it on colllab ?

Thank you!

**Comment by jaanaltosaar** [comment]
I haven't tested it with colab. Can you split the code according to the whitespace into separate cells in colab? Then link to the colab notebook, highlighting which cell gives an error?

**Comment by allaabdella2** [comment]
Yes, sure.
I’ll do it and send it to you.

**Comment by ExeCuteRunrunrun** [comment]
Hi, Altosaar and alla15747,
Did you figure out the problem ? I encountered the same error using a jupyter notebook with a gpu. The error occurs at the last iter of training and says : 
"Iteration: 99000 ELBO: -82.887 s/iter: 2.607e-03
An exception has occurred, use %tb to see the full traceback.
SystemExit"
I checked the log directory, it contains all jpg pictures until the 99000th one.
Many thanks to you !
